packages/neuron-wallet/src/controllers/sudt.ts

```diff
@@ -0,0 +1,36 @@
+import LiveCellService from "services/live-cell-service";
+import env from 'env'
+import Script, {ScriptHashType} from "models/chain/script";
+import parseSUDTTokenInfo from "utils/parse_sudt_token_info";
+import {ResponseCode} from "utils/const";
+
+export interface TokenInfo {
+  tokenId: string
+  symbol: string
+  tokenName: string
+  decimal: string
+}
+
+export default class SUDTController {
```

Comment:
Please add unit tests for this controller as well.

Reply:
fix